lib/notification_service/notification_service/cli/commands/server_command.py

```diff
@@ -48,6 +48,9 @@ def make_log_dir_if_not_exist():
 def server_start(args):
     pid_file_path = os.path.join(notification_service.settings.NOTIFICATION_HOME,
                                  notification_service.settings.NOTIFICATION_PID_FILENAME)
+    if os.path.exists(pid_file_path):
```

Comment:
We can use `check_pid_exist` method to check if the process is actually running

Reply:
Thanks for your review, I'll add a check.